zk/contracts/l1_contracts.go

```diff
@@ -16,4 +16,5 @@ var (
 	AddNewRollupTypeTopic          = common.HexToHash("0xa2970448b3bd66ba7e524e7b2a5b9cf94fa29e32488fb942afdfe70dd4b77b52")
 	CreateNewRollupTopic           = common.HexToHash("0x194c983456df6701c6a50830b90fe80e72b823411d0d524970c9590dc277a641")
 	UpdateRollupTopic              = common.HexToHash("0xf585e04c05d396901170247783d3e5f0ee9c1df23072985b50af089f5e48b19d")
+	RollbackSequenceTopic          = common.HexToHash("0x1125aaf62d132d8e2d02005114f8fc360ff204c3105e4f1a700a1340dc55d5b1")
```

Comment:
I'd consider renaming it to `RollbackBatchesTopic` (https://github.com/0xPolygonHermez/zkevm-contracts/blob/a9b4f742f66bd4f3bcd98a3a188422480ffe0d4e/contracts/v2/PolygonRollupManager.sol#L328-L332), in order to be consistent with the smart contracts. WDYT?

Reply:
Good idea, will change.